### PR TITLE
Companion for Substrate#9547

### PR DESCRIPTION
**Pull request by bkchr**

### CODE REVIEW

**Comment by bkchr** [comment]
bot merge cancel

File: node/client/Cargo.toml [hunk 1]
```diff
@@ -29,7 +29,7 @@ sc-service = { git = "https://github.com/paritytech/substrate", branch = "master
 
 pallet-mmr-primitives = { git = "https://github.com/paritytech/substrate", branch = "master" }
 
-beefy-primitives = { git = "https://github.com/paritytech/grandpa-bridge-gadget", branch = "master" }
+beefy-primitives = { git = "https://github.com/paritytech/grandpa-bridge-gadget", branch = "bkchr-upgrade-jsonrpc" }
```
Comment (niklasad1) [inline node/client/Cargo.toml:32]: shouldn't these be `master`?

Comment (bkchr) [reply node/client/Cargo.toml]: This was to make the companion happy. It is now master

**Review by niklasad1 — APPROVED** [review]

**Comment by bkchr** [comment]
bot merge

**Pull request merged**